setup.py

```diff
@@ -32,6 +32,8 @@ def get_readme():
     packages=find_packages(),
     zip_safe=False,
     install_requires=[
+        # avoid django>=2.1.0,<2.1.5 due to CVE-2019-19118
```

Comment:
```suggestion
        # avoid django>=2.1.0,<2.1.15 due to CVE-2019-19118
```

Also should there be something about 2.2.0 - 2.2.8?

Reply:
Good catch. 

Github's publications on this CVE are confusing. In the original alert it said it was fixed in 2.1.5, and that version is also listed as the "patched version" in the right-hand column of https://github.com/advisories/GHSA-hvmf-r92r-27hr:

![image](https://user-images.githubusercontent.com/464726/70253058-7d959900-1750-11ea-8932-f6e26ad9163f.png)

However, [top link](https://nvd.nist.gov/vuln/detail/CVE-2019-19118) on that page says 2.1.15, and the [commit where it was fixed](https://github.com/django/django/commit/103ebe2b5ff1b2614b85a52c239f471904d26244) was made long after the 2.1.5 release date (January 4, 2019).

Reply:
> Github's publications on this CVE are confusing

I don't think they're confusing, just wrong/typo. They actually got it right in their blurb about it (which is where I saw it). I wonder if there's a way to file a bug report on that?